download_data.py

```diff
@@ -58,3 +59,20 @@ def download_data(
         dataset_name=args.dataset_name,
         dataset_subset=args.dataset_subset,
         dataset_root_dir=args.dataset_dir)
+
+    # Initialize, setup, and parse the argument parser
+    parser = ArgumentParser(prog="Data Downloader")
+
+    parser.add_argument("--dataset-name", type=str, required=True,
+        help="Hugging Face dataset name. Should also set --dataset-subset.")
+    parser.add_argument("--dataset-subset", type=str, required=True,
+        help="Subset/config to use for Hugging Face dataset.")
+    parser.add_argument("--dataset-dir", type=str, required=True,
+        help="Path to directory in which Hugging Face datasets are downloaded.")
+
+    args = parser.parse_args()
+
+    download_data(
```

Comment:
A simpler way to do this is found in train_model.py --> train_model(**vars(args))

**vars(args) works well. However, this is not necessary.